src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/AssemblyResultSet.java

```diff
@@ -501,13 +501,17 @@ private void updateReferenceHaplotype(final Haplotype newHaplotype) {
      *
      * <p/>
      * The result is sorted incrementally by location.
-     *
+     * @param maxMnpDistance Phased substitutions separated by this distance or less are merged into MNPs.  More than
```

Comment:
This argument can't be `<=0`, right?  If I'm right, can you update the docs?  Any public methods should check the value with `ParamUtils.isPositive(...)` call.

Reply:
done